README.md

```diff
@@ -118,19 +118,19 @@ It is important for any code change to pass both unit and end-to-end tests. This
 
 ### Run End-to-End (E2E) Tests
 
-1. Set up the Nitro node
+1. Set up the Nitro test-node
 
    1. Download and install [Docker](https://www.docker.com/)
 
-   2. Check out the [Nitro repo](https://github.com/OffchainLabs/nitro)
+   2. Check out the [Nitro test-node repo](https://github.com/OffchainLabs/nitro-testnode)
 
    3. Run: (Make sure your Docker App is running)
 
       ```bash
       $ ./test-node.bash --init
       ```
 
-   4. When the Nitro node is up and running you should see logs like `sequencer_1` and `staker-unsafe_1` in the terminal. This can take up to 10 minutes
+   4. When the Nitro test-node is up and running you should see logs like `sequencer_1` and `staker-unsafe_1` in the terminal. This can take up to 10 minutes
```

Comment:
i don't think this needs any changes, it is a Nitro node
but for consistency it shouldn't have an hyphen

and add a full stop after "10 minutes"?